bartender/local_plugins/logger.py

```diff
@@ -20,30 +21,45 @@ def __getattr__(self, attr):
 
 
 # Since we are imitating the logging module, we will allow camel case method names
-def getPluginLogger(name, formatted=True, log_directory=None):
+def getPluginLogger(name, formatted=True, log_directory=None, log_name=None):
     """
     Get a Plugin Logger. If formatted is set to true, do no special formatting.
     Otherwise, make the logger log to something semi-pretty
 
-    :param name:
-    :param formatted:
-    :return:
+    Args:
+        name (str): The name of the logger to create
+        formatted (str / bool): The formatting to apply to the logger
```

Comment:
Maybe instead of using `formatted` as a `str` or a `bool` we should just pass in the format string itself?

Reply:
Oh good idea